src/copygb2/copygb2.f

```diff
@@ -1555,41 +1442,26 @@ SUBROUTINE INTGRIB1(K1F,KGDS1F,K2F,KGDS2F,MRL,MRO,
           ENDIF
         ENDIF
       ENDIF
-C - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
       END
-C-----------------------------------------------------------------------
+
+!> Return the length of a filled grid.
+!>
+!> Given a grid description section (in w3fi63 format), return the grid
```

Comment:
w3fi63() ?

Reply:
Are you suggesting parens? I don't know if w3fi63 is a function...

I have no idea what it means. ;-)

Reply:
Yeah I was curious if it was a function